src/modules/app/containers/I18nProvider.js

```diff
@@ -1,14 +1,12 @@
 import i18n from 'i18next'
```

Comment:
Do we need the I18nProvider at all? If I can remember it right, the Wohnungsbörse should only be available in german

Reply:
Well, it's better to leave it there, so we can handle all ui-text-related stuff in the locales.json.

Reply:
okay :+1: